### DISCUSSION TITLE
Can it compare two md contents?

### DISCUSSION THREAD

**Discussion by lanrenbulan** [thephpleague/commonmark#1020] [Q&A]
When comparing two md contents, differences can be displayed?

like this:

```diff
-  hello
+ hello!
```

**Comment by colinodell** [comment]
You could use this library to parse two Markdown documents into abstract syntax trees, but you'd have to write the code to compare them yourself.  We also don't (yet) have a way to render ASTs back into Markdown, meaning you'd also need to imlpement your own code to render those differences.

But given that Markdown files are just text, you could simply `diff` them instead.
